spec/spec.md

```diff
@@ -302,46 +306,53 @@ authorize
         “icon”: “<dapp_icon_relative_path>”,
         “name”: “<dapp_name>”,
     },
+    "cluster": "<cluster>",
 }
 ```
 
 where:
 
-- `dapp_uri`: (Optional) a URI representing the web address associated with the dapp endpoint making this authorization request. If present, it must be an absolute, hierarchical URI.
-- `dapp_icon_relative_path`: (Optional) a relative path (from dapp_uri) to an image asset file of an icon identifying the dapp endpoint making this authorization request
-- `dapp_name`: (Optional) the display name for this dapp endpoint
+- `identity`: a JSON object, containing:
+  - `uri`: (optional) a URI representing the web address associated with the dapp endpoint making this authorization request. If present, it must be an absolute, hierarchical URI.
+  - `icon`: (optional) a relative path (from `uri`) to an image asset file of an icon identifying the dapp endpoint making this authorization request
+  - `name`: (optional) the display name for this dapp endpoint
+- `cluster`: (optional) if set, the Solana network cluster with which the dapp endpoint intends to interact; supported values include `mainnet-beta`, `testnet`, `devnet`. If not set, defaults to `mainnet-beta`.
```

Comment:
`cluster` is Solana-specific, would we consider `network` here? If the API later comes to allow a [chain ID](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-2.md) of some kind, `network` could still be multi-chain.

Reply:
Same answer as my previous comment re: `options`.

IMO, every chain is going to have a few unique things, which are best captured in chain-specific companion specs. For now, since v1.0 is a Solana-only spec, they are encoded right into the main spec, but I envision this cluster field as being one of those.